tap/tlstapper/go_offsets.go

```diff
@@ -145,6 +146,16 @@ func getOffsets(filePath string) (offsets map[string]*goExtendedOffset, err erro
 		symEndingIndex := symStartingIndex + sym.Size
 
 		// collect the bytes of the symbol
+		textSectionDataLen := uint64(len(textSectionData) - 1)
+		if symEndingIndex > textSectionDataLen {
+			logger.Log.Errorf(
```

Comment:
i can see that we have now many crashes, does it mean that it will be printed as error now? if yes, I am wondering if it have to be error?

Reply:
We can print as `Infof` instead? It's caused by the lack of disassembly info in the binaries.

Reply:
or `Warningf` maybe?

Reply:
Changed it to `Warningf` with https://github.com/up9inc/mizu/commit/f56e1b135ba0109bf9cdfeb14d76fdd033d51af5